From d66445638b59a6510158b60d422e81d4b68cdeec Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Wed, 29 Mar 2017 18:34:10 +0800
Subject: [PATCH] arm: dts: rk3288-evb: add rockchip-relinquish-port quirk for
 ehci

This adds force abnormal ohci relinquish port owner
and back to ehci on rk3288 SoC.

Change-Id: I33be55c08762be7e8a239f741a8c8dbb28522306
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-evb.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rk3288-evb.dtsi b/arch/arm/boot/dts/rk3288-evb.dtsi
index 41b8eae3304c..1442b6179d57 100644
--- a/arch/arm/boot/dts/rk3288-evb.dtsi
+++ b/arch/arm/boot/dts/rk3288-evb.dtsi
@@ -530,6 +530,7 @@
 };
 
 &usb_host0_ehci {
+	rockchip-relinquish-port;
 	status = "okay";
 };
 
-- 
2.35.3

